software/controller/lib/core/nvparams.h

```diff
@@ -44,6 +55,18 @@ struct Structure {
   uint32_t cumulated_service{0};  // Cumulated power-ON time, stored in seconds.
                                   // May rollover after 136 years
   VentParams last_settings = VentParams_init_default;  // Last settings seen by the vent
+
+  // The following 2 tables are used to roughly linearize the pinch valves output.  They were built
+  // by adjusting the pinch valve and monitoring the flow through the venturi tube. The entries
+  // should give pinch valve settings for a list of equally spaced flow rates.  The first entry
+  // should be the setting for 0 flow rate (normally 0) and the last entry should be the setting
+  // for 100% flow rate. The minimum length of the table is 2 entries.
+  std::array<float, pinch_valves_cal_size> blower_pinch_cal{0.0000f, 0.0410f, 0.0689f, 0.0987f,
```

Comment:
Initial default values should probably be naive linear "curves"? But again until we have better calibration routines, this is probably fine. TODO?

Reply:
At least during prototyping phase, initial default values should match the "typical" unit produced (whatever that is), in order for stuff to work even if we haven't specifically calibrated the exact unit.
In production however, I'd probably try to make it so a unit doesn't work if calibration hasn't been performed, either by zero-filling it or some other solution (have a boolean in there that we need to set to 1 during calibration?)